spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractBinderPropertiesAccessor.java

```diff
@@ -29,7 +29,7 @@
  *
  * @author Gary Russell
  */
-public abstract class AbstractBinderPropertiesAccessor implements BinderProperties {
+public abstract class AbstractBinderPropertiesAccessor {
```

Comment:
For now, I think we should rename to `AbstractBindingPropertiesAccessor`. We can consider splitting into multiple classes later.